frontend/wallet/src/render/StyleGuide.re

```diff
@@ -35,6 +36,8 @@ module CssElectron = {
     | `noDrag => `declaration(("-webkit-app-region", "no-drag"));
 };
 
+let headerHeight = `em(4.);
```

Comment:
Why not `rem`s?

Reply:
Yeah we should probably avoid using em's I just copied this over from the Header styles so we can use it in other places but yeah not the best place for this to live, probably better to have like a Spacing module that this lives in with everything in rem or px.